core/output/telegram.js

```diff
@@ -0,0 +1,43 @@
+var options = {
+    // Used by the monkey patch to node-telegram-bot-api
+    stopPolling: true
+};
+var TelegramBot = require.safe('node-telegram-bot-api');
+// Adds the missing stopPolling() method
+// https://github.com/yagop/node-telegram-bot-api/pull/51#issuecomment-217395990
+require.safe('monkey-patches-node-telegram-bot-api')(TelegramBot, options);
+
+var bot = null,
+    shim = require('../shim.js'),
+    api = null;
+
+sendMessage = function(message, thread, opts) {
+    bot.sendMessage(thread, message, opts);
+};
+
+exports.start = function(callback) {
+    var token = exports.config.token;
+    bot = new TelegramBot(token, {
+        polling: true
+    });
+
+    api = shim.createPlatformModule({
+        sendMessage: sendMessage,
```

Comment:
@phillipadsmith this one can be ignored. However send message is currenly a global when it should be a local (change `sendMessage = function` to `var sendMessage = function`)